extensions/unsplash/package.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "unsplash",
+  "title": "Unsplash",
+  "description": "Search images or collections on Unsplash, download, copy or set them as background without leaving Raycast!",
+  "icon": "command-icon.png",
```

Comment:
Can you provide a light/dark icon? Otherwise this won't look good in different themes. You can find more info [here](https://developers.raycast.com/information/manifest).